src/components/Code/Code.js

```diff
@@ -41,8 +41,57 @@ const Pre = styled.pre`
   font-size: ${props => props.theme.fontSizes.small};
 `;
 
+const PaddedLine = styled.div`
+  &:not(:last-child) {
+    padding-bottom: 8px;
+  }
+
+  /* required to make jsx children work with adding '\n' in multiLine */
+  & > div {
+    display: inline;
+  }
+`;
+
 const trimString = node => (isString(node) ? trim(node) : node);
 
+const formatSingleCommand = children =>
+  isFunction(children) ? children() : trimString(children);
+
+function formatMultiString(string) {
+  return trim(string)
+    .split('\n')
+    .map((line, i) => (
+      <PaddedLine key={i}>
```

Comment:
Passing indices as keys is not recommendable - how about `key={line}`?

Reply:
Can do, not sure if there are any implications of the key being an arbitrary string length/content wise? 
The other key in `formatMultiCommand` is also index but we don't have a unique identifier such as line, not sure what the usual form is in such a situation? 
While not ideal, using index only really becomes as issue when performance is mission critical which I wouldn't say it was in this component.

Reply:
> Can do, not sure if there are any implications of the key being an arbitrary string length/content wise?
seems to be fine and is escaped by react so no worries about user input 🎉